volatility3/framework/symbols/linux/__init__.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, *args, **kwargs) -> None:
 class LinuxUtilities(interfaces.configuration.VersionableInterface):
     """Class with multiple useful linux functions."""
 
-    _version = (2, 2, 0)
+    _version = (2, 1, 0)
```

Comment:
The version should go up, not down and given you're removing an API (get_inode_metadata), it will mean the version needs to go to `(3,0,0)` and will break all the existing linux plugins.  Might be worth leaving it, but including a `DecprecationException` so people know not to use it in the future?

Reply:
This was only in one commit in the PR, not the PR as a whole, sorry for the noise.